src/features/machines/depositUIMachine.ts

```diff
@@ -255,6 +268,13 @@ export const depositUIMachine = setup({
         context.formValues.network === BlockchainEnum.SOLANA
       )
     },
+    isDepositTurbochainRelevant: ({ context }) => {
+      return (
+        context.balance + context.nativeBalance >=
```

Comment:
I don't remember if I've asked you already, but why we add balance and native balance?

Reply:
This is only for the Near network, because if the user chooses Near for the deposit, then in this case his available balance is Near plus wNear. In this case it is a mistake, I will fix it.

Reply:
Also I'll do correction for `isDepositEVMRelevant` and `isDepositSolanaRelevant`